website/core/RemarkablePlugins.js

```diff
@@ -142,7 +127,6 @@ function ReactNativeWebPlayer(md) {
     if (paramsString) {
       hash += `&${paramsString}`;
     }
-    let params = parseParams(paramsString);
```

Comment:
This line is fine, the `parseParams` below in iframe `height` should be replaced with `params` variable.